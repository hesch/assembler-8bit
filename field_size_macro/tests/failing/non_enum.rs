#[macro_use]
extern crate field_size_macro;

#[derive(FieldSize)]
struct Test;
