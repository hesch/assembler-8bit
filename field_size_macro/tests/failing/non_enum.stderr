error: proc-macro derive panicked
 --> $DIR/non_enum.rs:4:10
  |
4 | #[derive(FieldSize)]
  |          ^^^^^^^^^
  |
  = help: message: FieldSize only works on Enums
